### ISSUE TITLE
Partially init attr

### ISSUE THREAD

**Issue by JZubero** [ErikSchierboom/knockout-pre-rendered#43]
I have two attributes to bind. One has an initial value and is a regular observable, the other one has no initial value since it is a computed observable.

```html
<div data-bind="init, attr: { 'data-id': id, 'data-index': index }" data-id="23"> ... </div>
```
Part of the view model:

```javascript
function MemberRowVM(id, surname) {
  var self = this;
  self.surname = ko.observable(surname);
  self.id = ko.observable(id);
  self.index = ko.computed(function() {
    return self.surname().strpos(0, 1);
  });
}
```

So I want `id` to be initialized and `index` not. With that version I end up with the following error:
> Uncaught TypeError: Unable to process binding "init: function (){return undefined }"

Is there a best practice way how to solve this?

**Comment by ErikSchierboom** [comment]
@JZubero This might be a bug. I'll look into it.

**Comment by ErikSchierboom** [comment]
@JZubero I've found and fixed the bug. A new version with this fix (0.7.2) has just been released. Thanks for reporting the bug and providing a repro!
